src/seedu/addressbook/data/person/Name.java

```diff
@@ -2,63 +2,95 @@
 
 import seedu.addressbook.data.exception.IllegalValueException;
```

Comment:
I think this is for activity 7. Next time, please send a separate PR for each activity.